tpp-app/src/home/components/CycleCalendarTabs.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import { createMaterialTopTabNavigator } from '@react-navigation/material-top-tabs';
+import CalendarScreen from '../pages/CalendarScreen';
+import CycleScreen from '../pages/CycleScreen';
+import Constants from 'expo-constants';
+
+
+const Tab = createMaterialTopTabNavigator();
+
+export default function HomeNavigator() {
+  return (
+      <Tab.Navigator 
+        style={{top: Constants.statusBarHeight}}
+        screenOptions={{
+          tabBarIndicatorStyle: {
+            height: 3,
+            backgroundColor: "#5A9F93",
+          },
+          tabBarLabelStyle: {
+            textTransform:'capitalize',
+            fontFamily: "Avenir",
+            fontSize: 20,
+            fontStyle: "normal",
+            fontWeight: "800",
+            lineHeight: 27,
+            letterSpacing: -0.02,
+            textAlign: "left",
+          },
+          tabBarStyle: {
+            height: "7%",
+            justifyContent: 'center'
+          }
+        }}
+      >
+        <Tab.Screen name="Calendar" component={CalendarScreen} />
```

Comment:
Can you add the Tab.Screen name to CalendarNavigator.js's STACK_SCREENS? So if we need to navigate to Calendar or Cycle from another place such as notifications using the constant will help avoid typo issues